src/gamma.jl

```diff
@@ -926,3 +926,22 @@ function logabsbinomial(n::T, k::T) where {T<:Integer}
     end
 end
 logabsbinomial(n::Integer, k::Integer) = logabsbinomial(promote(n, k)...)
+
+
+# this trickery is needed while the deprecated method in Base exists
+@static if !hasmethod(Base.binomial, Tuple{Number, Number})
+    import Base: binomial
```

Comment:
not clear if we want to commit type piracy here by extending `Base.binomial`, or just define our own `binomial` method that shadows Base?

Reply:
Deleted all of that. See commit c89743a.